schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+CREATE TABLE animals (
+ID INT,
+NAME  VARCHAR,
+DATE_OF_BIRTH DATE,
+ESCAPE_ATTEMPTS INT,
+NEUTERED BOOLEAN,
+WEIGHT_KG DECIMAL
```

Comment:
- [ ] [OPTIONAL] Kindly set the `id` as the primary key and set5 it to auto increment every time there is a new entry
- [ ] [OPTIONAL]  Kindly add character length on the varchar to limit the number of characters that can be inserted in that column